src/main/java/project/ui/pages/FormPageFour.java

```diff
@@ -0,0 +1,86 @@
+/*
+ * @(#) FormPageFour.java Copyright (c) 2019 Jala Foundation.
+ * 2643 Av. Melchor Perez de Olguin, Colquiri Sud, Cochabamba, Bolivia.
+ * All rights reserved.
+ *
+ * This software is the confidential and proprietary information of
+ * Jala Foundation, ("Confidential Information").  You shall not
+ * disclose such Confidential Information and shall use it only in
+ * accordance with the terms of the license agreement you entered into
+ * with Jala Foundation.
+ */
+
+package project.ui.pages;
+
+import core.utils.TrabajopolisUtils;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import project.ui.BasePage;
+
+/**
+ * FormPageFour class.
+ *
+ * @author John Salazar Pinto
+ * @version 1.0
+ */
+public class FormPageFour extends BasePage {
+
```

Comment:
Please delete this unnecessary first line(according to your other classes).

Reply:
done